console-subscriber/src/lib.rs

```diff
@@ -16,13 +16,69 @@ use tracing_core::{
     subscriber::{self, Subscriber},
     Metadata,
 };
-use tracing_subscriber::{layer::Context, registry::LookupSpan, Layer};
+use tracing_subscriber::{fmt, layer::Context, registry::LookupSpan, EnvFilter, Layer};
 
 mod aggregator;
 use aggregator::Aggregator;
 mod builder;
 pub use builder::Builder;
 
+/**
+Starts the console subscriber server on its own thread
+
+This function represents the easiest way to get started using
+tokio-console.
+
+## Configuration
+
+Tokio console subscriber is configured with sensible defaults for most
+use cases. If you need to tune these parameters, several environmental
+configuration variables are available:
+
+* `TOKIO_CONSOLE_RETENTION`: The number of seconds to accumulate
+  completed tracing data. Default: 60s.
+* `TOKIO_CONSOLE_BIND`: a HOST:PORT description, such as
+  localhost:1234 or similar. Default: 127.0.0.1:6669
+* `TOKIO_CONSOLE_PUBLISH_INTERVAL`: The number of milliseconds to wait
+  between sending updates to the console. Default: 1000ms (1s)
+* `RUST_LOG`: configure the tracing filter. Default: `tokio=trace`,
+  and any additional filtering directives will be appended to this
+  default. See [`EnvFilter`] for further information on the format
+  for this variable.
```

Comment:
take it or leave it: similarly, this might want to be a table?

Reply:
addressed in f2b6de51565b504f83b784f6de8f891fab929578